src/functionalTest/resources/request/casedata/ccd-callback-casedata-submit-case-about-to-submit.json

```diff
@@ -49,7 +49,8 @@
           "document_filename": null,
           "document_binary_url": null,
           "category_id": null
-        }
+        },
+        "comment": null
```

Comment:
Why not update the test to pass a doc with relevance so that we can verify that the new approach works end to end?